Toolkit/ArcGISToolkit/TimeSlider.swift

```diff
@@ -0,0 +1,2539 @@
+//
+// Copyright 2018 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import QuartzCore
+import ArcGIS
+
+// MARK: - Time Slider Control
+
+/*
+ Controls how labels on the slider are displayed.
+ */
+public enum TimeSliderLabelMode {
```

Comment:
@njarecha Could these types (`TimeSliderLabelMode`, `PlaybackDirection`, `LoopMode`, `PlaybackDirection`, and `Theme`) be nested under `TimeSlider`? They are documented as being specific to a time slider, but they are currently in the global namespace.

Reply:
fixed